lib/widgets/transaction_item.dart

```diff
@@ -198,31 +66,45 @@ class TransactionsItemState extends State<TransactionsItem> {
           transacionStatusCopy,
           textAlign: TextAlign.start,
           overflow: TextOverflow.ellipsis,
-          style: theme.textTheme.bodySmall,
+          style: theme.textTheme.bodyMedium,
         ),
       ],
     );
   }
 
-  @override
-  Widget build(BuildContext context) {
+  Color? getAmountColor(String prefix) {
     final theme = Theme.of(context);
     final extendedTheme = theme.extension<ExtendedTheme>()!;
-    String label;
-    String address;
-    TransactionType type = widget.transaction.type;
-
-    if (type == TransactionType.value_transfer) {
-      label = getTransactionLabel(externalAddresses, internalAddresses,
-          widget.transaction.vtt!.inputs, singleAddressAccount, context);
-      address = getTransactionAddress(label, widget.transaction.vtt!.inputs,
-          widget.transaction.vtt!.outputs);
+    int _lockedValue = transactionUtils!.lockedValue();
+    if (_lockedValue > 0) {
+      return WitnetPallet.mediumGrey;
+    } else if (prefix == '-') {
+      return extendedTheme.txValueNegativeColor;
+    } else if (prefix == '+') {
+      return extendedTheme.txValuePositiveColor;
     } else {
-      // TODO(#542): set stake and unstake transaction label when feature is supported
-      label = localization.from;
-      address = 'Mint';
+      return theme.textTheme.bodyLarge!.color;
     }
-    int _lockedWit = lockedValue(widget.transaction);
+  }
+
+  Widget buildTransactionValue(label) {
+    final theme = Theme.of(context);
+    TransactionValue transactionValue = transactionUtils!.getTransactionValue();
+
+    return Text(' ${transactionValue.prefix} ${transactionValue.amount}',
+        style: theme.textTheme.bodyLarge?.copyWith(
```

Comment:
The time should be smaller than the amount. I think the amount fontSize should be same size as the address size or bigger.
![Screenshot 2024-10-07 at 16 02 56](https://github.com/user-attachments/assets/d5e6cf9c-4261-46e3-8031-f4d51c5a9c93)